main.py

```diff
@@ -1,7 +1,24 @@
 import init_django_orm  # noqa: F401
+from db.models import Genre, Actor
 
-from django.db.models import QuerySet
+def main() -> None:
+    Genre.objects.create(name="Western")
+    Genre.objects.create(name="Action")
+    Genre.objects.create(name="Dramma")
 
+    Actor.objects.create(first_name="George", last_name="Klooney")
+    Actor.objects.create(first_name="Kianu", last_name="Reaves")
+    Actor.objects.create(first_name="Scarlett", last_name="Keegan")
+    Actor.objects.create(first_name="Will", last_name="Smith")
+    Actor.objects.create(first_name="Jaden", last_name="Smith")
+    Actor.objects.create(first_name="Scarlett", last_name="Johansson")
 
-def main() -> QuerySet:
-    pass
+    Genre.objects.filter(name="Dramma").update(name="Drama")
+    Actor.objects.filter(first_name="George").update(last_name="Clooney")
+    Actor.objects.filter(first_name="Kianu").update(first_name="Keanu", last_name="Reeves")
+
+    Genre.objects.filter(name="Action").delete()
+    Actor.objects.filter(first_name="Scarlett").delete()
```

Comment:
The deletion of actors with the first name 'Scarlett' will remove all such entries, including 'Scarlett Johansson'. If you intend to keep some entries, consider refining the filter criteria.